hysds/recognize.py

```diff
@@ -136,14 +136,6 @@ def getLevel(self):
         else:
             return self.recognized['level'].format(**self.group_dict)
 
-    def getType(self):
```

Comment:
@DustinKLo: there may be some confusion here. This function returns the dataset type as defined in the `datasets.json`, e.g. https://github.com/hysds/hysds/blob/develop/configs/datasets/datasets.json#L7. So this should remain there.